pulsar-common/src/main/java/org/apache/pulsar/common/util/FutureUtil.java

```diff
@@ -69,6 +70,32 @@ public static <T> CompletableFuture<List<T>> waitForAll(Stream<CompletableFuture
                 })));
     }
 
+    /**
+     * Make {@param dest} completed as {@param src}.
```

Comment:
Please add better description. 
`completeAs` might not be the best possible name for this method. `completeAfter` would be a better name.

Reply:
Renamed to `completeAfter`,  and modified the description.